### PR TITLE
Add git ignore for bot templates

### PR DESCRIPTION
**Pull request by YitongFeng-git**

### CODE REVIEW

File: bot/js/.gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# dependencies
+node_modules/
+package-lock.json
```
Comment (hund030) [inline bot/js/.gitignore:3]: Do we add package-lock.json to git ignore?

Comment (YitongFeng-git) [reply bot/js/.gitignore:3]: I think so, it will generated when npm install

Comment (YitongFeng-git) [reply bot/js/.gitignore:3]: Synced with zhijie, it's better to not put it in git ignore

File: bot/ts/.gitignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# dependencies
+node_modules/
+
+# misc
+.env
```
Comment (henzhang-ms) [inline bot/ts/.gitignore:5]: Nit: ts would have build files.

Comment (YitongFeng-git) [reply bot/ts/.gitignore:5]: add lib/

**Review by henzhang-ms — APPROVED** [review]

**Pull request merged**